clouddriver-artifacts/clouddriver-artifacts.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
   implementation "com.oracle.oci.sdk:oci-java-sdk-core"
   implementation "com.squareup.okhttp:okhttp"
   implementation "com.sun.jersey:jersey-client:1.9.1"
+  implementation 'commons-io:commons-io:2.15.1'
```

Comment:
Why do we need this?  And why this version?

Reply:
oci-java-sdk-bom:2.32.0 is the last version to use common-io:commons-io dependency. oci-java-sdk-bom:2.33.0 doesn't have it. 
So when oci-java-sdk-bom is upgraded, one of the java files complained about missing common-io library.

Reply:
It's the latest version.

Reply:
```
clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/CredentialReader.java:23: error: package org.apache.commons.io does not exist
import org.apache.commons.io.FileUtils;
```